From f662280b26eb1549890e88e8e9085f1937d7962d Mon Sep 17 00:00:00 2001
From: Tim Orling <tim.orling@konsulko.com>
Date: Mon, 28 Feb 2022 07:54:03 -0800
Subject: [PATCH] setup.py: switch to setuptools

In Python 3.10, 'distutils' is deprecated and is slated for removal in
Python 3.12.

Switch from distutils.core to setuptools. This also allows the 'wheel'
binary archive format to be built with 'setup.py bdist_wheel'.

Upstream-Status: Submitted [https://github.com/bb4242/sdnotify/pull/7]

Signed-off-by: Tim Orling <tim.orling@konsulko.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 433222c..32b5bd6 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-from distutils.core import setup
+from setuptools import setup
 
 VERSION='0.3.2'
 
-- 
2.30.2

